network-api/networkapi/wagtailpages/templates/wagtailpages/blocks/typeform_block.html

```diff
@@ -0,0 +1,21 @@
+<script
```

Comment:
Would this do "fun things(tm)" if we have multiples of this block on a single page?